improver_tests/acceptance/SHA256SUMS

```diff
@@ -700,37 +700,42 @@ c798e3530eac9a45305711d5e91daa56d5005b826ce85bb48e2889e328ea965c  ./wxcode-modal
-79d05a619d981811398bf6dd3a3bc235f21d9a30b998000ad7f45617a0bb30dc  ./wxcode/global/probability_of_lwe_thickness_of_precipitation_amount_above_threshold.nc
-9a4dbc8c6f1958efde70f618cf0aa6be858ce9f8b88ff445e7ef98905b13ee34  ./wxcode/global/probability_of_lwe_thickness_of_precipitation_amount_in_vicinity_above_threshold.nc
-356d1469e500c19a7465fc1c4305ba394b27843557ae21fa653fc8d29091c7c8  ./wxcode/global/probability_of_lwe_thickness_of_sleetfall_amount_above_threshold.nc
-cf2cebc9740adf7b3754bb6fc19ec6b92dfa0f0d0a266da752bb0c28eb44f440  ./wxcode/global/probability_of_lwe_thickness_of_snowfall_amount_above_threshold.nc
-0d541cc310787f23979c3d9b43fd66d5188adc8bb62fc84dc40c04b2ee030a36  ./wxcode/global/probability_of_shower_condition_above_threshold.nc
-70544bcbc20a173c990a91cad94c0f70d83906bfe8b9b6bd7e65cd79d8374ac5  ./wxcode/global/probability_of_thickness_of_rainfall_amount_above_threshold.nc
-6bc02d671b45d3f56deee2fe6002c7c642115e5f481ec0d51c28ed7a9cef6b8a  ./wxcode/global/probability_of_visibility_in_air_below_threshold.nc
-59f399b9944948af2f4da472756ec91ca281fa19f28ec616aa50e429cabd760b  ./wxcode/native_units/probability_of_lightning_flashes_per_unit_area_in_vicinity_above_threshold.nc
-a9bf67edcc91d6e75af07a24ca5b779ea5d4014ce66a3406aa5a89a1ab7c6865  ./wxcode/native_units/probability_of_low_and_medium_type_cloud_area_fraction_above_threshold.nc
-cb5c16a5fd656ee157b40659187a4742f36872622b21b729ed893679cba889b4  ./wxcode/native_units/probability_of_low_type_cloud_area_fraction_above_threshold.nc
-8121978e021117a08acae6782914f36329ac4d7cb956fc3a4a83f7572345c4ab  ./wxcode/native_units/probability_of_lwe_thickness_of_precipitation_amount_above_threshold.nc
-bf1f7b63fb5abf66375f162b371acf49f9ab7b327b4315bbaacb8d9ef95590f7  ./wxcode/native_units/probability_of_lwe_thickness_of_precipitation_amount_in_vicinity_above_threshold.nc
-4daea2d64e5b55507b7d0cda37642a08e265e89e5f72584845182e45ffa37eab  ./wxcode/native_units/probability_of_lwe_thickness_of_sleetfall_amount_above_threshold.nc
-8699b5267ecfdba1259d4b3d0a9a7ef33b75838d5ded75d041d48938a556a5fa  ./wxcode/native_units/probability_of_lwe_thickness_of_snowfall_amount_above_threshold.nc
-a5e061dd1d1b476f4e9bf2d959a7ee267ceaeb41bc0ddb4f95e0208b2c267a37  ./wxcode/native_units/probability_of_shower_condition_above_threshold.nc
-0ebf269ce0bd8bdb87322589163ec266c7e3b4fa62133e6db840910d942ad41e  ./wxcode/native_units/probability_of_thickness_of_rainfall_amount_above_threshold.nc
-9831c847f94f68b1a4917e9d68f123d54fc9f87447ff6d34fddf71bfa4737117  ./wxcode/native_units/probability_of_visibility_in_air_below_threshold.nc
-10a3502534d286f0abb732bc4a4bb0b9e494a073b2d572693e080d5e4633d66d  ./wxcode/wx_decision_tree_1h.json
-f2b140ab62972b5fe5e52ddef4550a9fde440da825070999c0a51d08f10d060f  ./wxcode/wx_decision_tree_3h.json
+5a3a808f0cc7952e6cc4225d09136276f685c5fdd6798a795c0430aa65ca71fa  ./wxcode/basic/kgo.nc
```

Comment:
Just a couple of queries regarding these acceptance tests.

1. Based on [this comment](https://github.com/metoppv/mo-blue-team/issues/92#issue-1016463122), it seems like the intention is not to include "in_vicinity" within the cell method for the vicinity diagnostics. For the inputs to the weather symbols, it seems like they do not have "in_vicinity" in the cell method, which seems correct but ​`interpret-metadata`​ seems to think that they should. Will just behaviour of ​`interpret-metadata`​ be corrected, or has been corrected by one of your PRs e.g. https://github.com/metoppv/improver/pull/1632? 

2. ​`interpret-metadata`​ warns about the ​`mosg__model_run​` attribute, which is present on some but not all of the inputs. Should this attribute be present?

Reply:
OK, I've now realised that the `mosg__model_run` attribute is potentially expected after model blending as model provenance information following https://github.com/metoppv/improver/pull/1569. The presence of the `mosg__model_run` attribute on these inputs is, therefore, fine.

Reply:
Given that `mosg__model_run` is an accepted attribute, potentially `interpret_metadata` should be updated to reflect this.